src/sider/sykmeldinger/sykmelding/sykmeldingOpplysninger/Nokkelopplysning.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactNode } from "react";
+import { FieldReadOnly } from "@/components/FieldReadOnly";
+
+interface NokkelopplysningProps {
```

Comment:
Funker bare Props? 😊
```suggestion
interface Props {
```